packages/query/src/query-engine/index.ts

```diff
@@ -59,13 +63,68 @@ export class QueryEngine {
 			QueryEngine.initialize();
 		}
 
-		const data = await QueryEngine.apolloClient.query( {
-			query: GetGlobalStylesDocument,
-			fetchPolicy: 'network-only', // @todo figure out a caching strategy, instead of always fetching from network
-			errorPolicy: 'all',
-		} );
+		try {
+			const data = await QueryEngine.apolloClient.query( {
+				query: GetGlobalStylesDocument,
+				fetchPolicy: 'network-only', // @todo figure out a caching strategy, instead of always fetching from network
+				errorPolicy: 'all',
+			} );
+
+			return parseGlobalStyles( data );
+		} catch ( error ) {
+			if ( error instanceof ApolloError ) {
+				// If there are graphQLErrors log them.
+				error.graphQLErrors.forEach( ( graphQLError ) => {
+					Logger.error( graphQLError.message );
+				} );
+
+				// If there are clientErrors log them.
+				error.clientErrors.forEach( ( clientError ) => {
+					Logger.error( clientError.message );
+				} );
+
+				// If there are protocolErrors log them.
+				error.protocolErrors.forEach( ( protocolError ) => {
+					Logger.error( protocolError.message );
+				} );
```

Comment:
Refactor this a utility function named `logGraphQLErrors` and reuse in both of the queries.